From 97a54ffa0d4972be6980d7486b02a7d0262b253f Mon Sep 17 00:00:00 2001
From: Wangming Shao <shaowangming@h-partners.com>
Date: Tue, 5 Jul 2022 17:56:22 +0800
Subject: [PATCH] scsi: hisi_sas: Change the debugfs parameters dump count
 default value
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 97a54ffa0d4972be6980d7486b02a7d0262b253f
Modified-by-SEL: No


driver inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I5DKH2

-----------------------------------------------------------------------------

If the default hisi_sas_debugfs_dump_count is 50, a large amount of memory
is occupied.
Therefore, to avoid high memory usage, reduce the
hisi_sas_debugfs_dump_count from 50 to 1.

Signed-off-by: Wangming Shao <shaowangming@h-partners.com>
Reviewed-by: Jason Yan <yanaijie@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 drivers/scsi/hisi_sas/hisi_sas_main.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/scsi/hisi_sas/hisi_sas_main.c b/drivers/scsi/hisi_sas/hisi_sas_main.c
index 4cbe8711b6d4..a1c6a67da132 100644
--- a/drivers/scsi/hisi_sas/hisi_sas_main.c
+++ b/drivers/scsi/hisi_sas/hisi_sas_main.c
@@ -2852,13 +2852,13 @@ EXPORT_SYMBOL_GPL(hisi_sas_remove);
 #if IS_ENABLED(CONFIG_SCSI_HISI_SAS_DEBUGFS_DEFAULT_ENABLE)
 #define DEBUGFS_ENABLE_DEFAULT  "enabled"
 bool hisi_sas_debugfs_enable = true;
-u32 hisi_sas_debugfs_dump_count = 50;
 #else
 #define DEBUGFS_ENABLE_DEFAULT "disabled"
 bool hisi_sas_debugfs_enable;
-u32 hisi_sas_debugfs_dump_count = 1;
 #endif
 
+u32 hisi_sas_debugfs_dump_count = 1;
+
 EXPORT_SYMBOL_GPL(hisi_sas_debugfs_enable);
 module_param_named(debugfs_enable, hisi_sas_debugfs_enable, bool, 0444);
 MODULE_PARM_DESC(hisi_sas_debugfs_enable,
-- 
2.34.1

